### PR TITLE
Factor out a Concepts section

### PR DESCRIPTION
**Pull request by dcreager**
This updates the spec to follow the same basic structure as Reporting, with a separate top-level Concepts section, and separate sections on Policy Delivery and Report Delivery.

### CODE REVIEW

**Comment by dcreager** [comment]
@igrigorik, do you want to review this before I merge it?  It *should* just be a refactoring, with no changes to the logic.

File: index.html [hunk 1]
```diff
@@ -7,6 +7,8 @@
   <script src='https://www.w3.org/Tools/respec/respec-w3c-common' async class='remove'></script>
   <script class='remove'>
   var respecConfig = {
+    wg: "Web Platform Incubator Community Group",
```
Comment (igrigorik) [inline index.html]: We did formally agree to adopt Reporting and NEL into webperf WG. We should do the move and update the WG reference.

@plehegar @siusin @yoavweiss can you help coordinate the transfer?

File: index.html [hunk 2]
```diff
@@ -133,24 +136,41 @@ <h2>Dependencies</h2>
...
+            <li><dfn data-cite="!HTML#navigate" data-lt="navigation">navigate</dfn></li>
             <li><dfn data-cite="!HTML#navigator.online"><code>navigator.onLine</code></dfn></li>
-            <li>resource <dfn data-cite="!HTML#origin">origin</dfn></li>
+            <li>resource <dfn data-cite="!HTML#origin" data-lt="origins">origin</dfn></li>
           </ul>
         </dd>
         <dt>HTTP</dt>
         <dd>
-          <p>The following terms are defined in the HTTP specification: [[!RFC7231]]</p>
+          <p>The following terms are defined in the HTTP specification: [[!RFC7230]], [[!RFC7231]], [[RFC7234]]</p>
```
Comment (igrigorik) [inline index.html]: Any particular reason for non-normative ref to 7234?

File: index.html [hunk 3]
```diff
@@ -212,293 +234,583 @@ <h2>Dependencies</h2>
...
+
+      <section>
+        <h2>The <code>success_fraction</code> member</h2>
+
+        <p>
+        The OPTIONAL <dfn><code>success_fraction</code></dfn> member defines the
+        <a>sampling rate</a> that should be applied to reports about
+        <a>successful</a> <a>network requests</a> for this origin.  If present,
+        its value MUST be a number between <code>0.0</code> and
+        <code>1.0</code>, inclusive; any other value will result in a parse
```
Comment (igrigorik) [inline index.html]: Parse error? As in, the whole registration is aborted? Perhaps, "any other value will be ignored"?

File: index.html [hunk 4]
```diff
@@ -212,293 +234,583 @@ <h2>Dependencies</h2>
...
-          <dd>The connection was aborted due to an HTTP protocol error</dd>
+      <section>
+        <h2>The <code>report_to</code> member</h2>
 
-          <dt>http.response.invalid</dt>
-          <dd>Response is empty, has a content-length mismatch, has improper encoding, and/or other conditions that prevent user agent from processing the response</dd>
+        <p>
+        The <dfn><code>report_to</code></dfn> member specifies the <a>endpoint
+          group</a> that reports for this <a>NEL policy</a> will be sent to.
+        The <a>report_to</a> member is REQUIRED to register a <a>NEL policy</a>,
```
Comment (igrigorik) [inline index.html]: Is the REQUIRED necessary here? In reporting we have a [fallback to "default"](https://wicg.github.io/reporting/#id-member), any reason why we shouldn't extend the same behavior here?

File: index.html (hunk 1 above)
Comment (dcreager) [reply index.html]: Updated the references

File: index.html (hunk 2 above)
Comment (dcreager) [reply index.html]: Ah, good catch!  The text below that talks about cached requests not being "network requests" used to be in a note paragraph, which I think would've made any references non-normative?  But I made that paragraph normative, so the reference should be too.

Fixed

File: index.html (hunk 3 above)
Comment (dcreager) [reply index.html]: This is consistent with what we do in Reporting.  If you leave an optional field out, we provide a suitable default, but if you provide the field with an invalid value, we consider the entire policy invalid.  If we want to change this I think we should update Reporting at the same time, and it should be in a separate PR.

File: index.html (hunk 4 above)
Comment (dcreager) [reply index.html]: I like that!  I'd like to keep this as a refactoring PR, so I opened #84 to make sure to change this in a follow-on PR.

Comment (igrigorik) [reply index.html]: 👍

File: index.html (hunk 3 above)
Comment (igrigorik) [reply index.html]: Hmm, I'm generally in favor of "be liberal in what you accept" principle.. unless the fallback runs the risk of some security / other exposure. In this case, seems like the fallback would be "0.0", which is ~safe?

File: index.html (hunk 1 above)
Comment (yoavweiss) [reply index.html]: Happy to help on the WICG side. 

@dcreager - will you be able to send out an [intent to migrate](http://wicg.github.io/admin/intent-to-migrate.html) to https://discourse.wicg.io?

@siusin /@plehegar - what's the procedure from the W3C side of things? Do I just transfer the repo to the W3C org?

Comment (siusin) [reply index.html]: @sideshowbarker has the admin access to both orgs (W3C/WICG), please give him a ping when the intent to migrate has been sent out.

Comment (dcreager) [reply index.html]: Just filed an [intent to migrate](https://discourse.wicg.io/t/intent-to-migrate-reporting-and-network-error-logging/2864?u=dcreager).  Please take a look and let me know if I missed anything!

Comment (dcreager) [reply index.html]: Also opened wicg/reporting#100 to track a similar move for the Reporting spec.

Comment (yoavweiss) [reply index.html]: Pinged @sideshowbarker on IRC to try to get him to move the repos over

Comment (dcreager) [reply index.html]: Repos have been moved over

**Pull request merged**